src/NLog/LogFactory.cs

```diff
@@ -1196,6 +1212,12 @@ internal LogFactory LoadConfiguration(string configFile, bool optional)
             return this;
         }
 
+        internal LoggingConfiguration CreateConfig(string configFile)
```

Comment:
Method extraction is no loading needed, and I like calling `_configLoader.Load` better than the name `CreateConfig`.